website/docs/r/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -77,6 +81,8 @@ Review the argument references that you can specify for your resource.
   **NOTE**: `id` and `subnet` are mutually exclusive.
 
 - `resource_group` - (Optional, Forces new resource, String) The resource group ID.
+- `security_groups` - (Optional, list) The security groups to use for this endpoint gateway. If unspecified, the VPC's default security group is used.
+    **NOTE:** either of ibm_is_security_group_target resource or security_groups attribute should be used, can't be user together. 
```

Comment:
**NOTE:** either of `ibm_is_security_group_target` resource or `security_groups` attribute should be used, both can't be use together.